python/lsst/obs/base/ingest.py

```diff
@@ -423,6 +635,20 @@ def prep(self, files, *, pool: Optional[Pool] = None, processes: int = 1) -> Ite
             pool = Pool(processes)
         mapFunc = map if pool is None else pool.imap_unordered
 
+        # Look for index files and read them
+        # There should be far fewer index files than data files
+        index_entries, files, good_index_files, bad_index_files = self.expandIndexFiles(files)
+        if bad_index_files:
+            self.log.info("Failed to read the following index files:"),
```

Comment:
Should these be `warn` or even `error`, to make them more obvious?

Reply:
I don't know. These match the log level reported from failing to read a data file. warn vs info vs debug is open to debate and I'm more than happy to change it all based on feedback from users. For now I'd like to keep it consistent with what is already in place.